lte/gateway/docker/mme/Dockerfile.ubuntu18.04

```diff
@@ -116,94 +123,131 @@ RUN git clone --recurse-submodules -b v1.15.0 https://github.com/grpc/grpc && \
         -DgRPC_CARES_PROVIDER=package \
         -DgRPC_SSL_PROVIDER=package \
         -DCMAKE_BUILD_TYPE=Release \
-        ../.. && \
+        .. && \
     make -j`nproc` && \
-    make install
+    make install && \
+    cd .. && rm -Rf _build
+
 
 ##### Prometheus CPP
-RUN git clone https://github.com/jupp0r/prometheus-cpp.git && \
-    cd prometheus-cpp && \
+RUN cd prometheus-cpp && \
+    # Moved git clone https://github.com/jupp0r/prometheus-cpp.git && \
     git checkout d8326b2bba945a435f299e7526c403d7a1f68c1f && \
     git submodule init && git submodule update && \
     mkdir _build && \
     cd _build/ && \
     cmake .. && \
     make -j`nproc` && \
-    make install
+    make install && \
+    cd .. && rm -Rf _build
 
 ##### Redis CPP
-RUN git clone https://github.com/cpp-redis/cpp_redis.git && \
-    cd cpp_redis && \
+RUN cd cpp_redis && \
+    # Moved git clone https://github.com/cpp-redis/cpp_redis.git && \
     git checkout bbe38a7f83de943ffcc90271092d689ae02b3489 && \
     git submodule init && git submodule update && \
-    mkdir build && cd build && \
+    mkdir _build && cd _build && \
     cmake .. -DCMAKE_BUILD_TYPE=Release && \
     make -j`nproc` && \
-    make install
+    make install && \
+    cd .. && rm -Rf _build
```

Comment:
Nitpick: In another Dockerfile we once discovered that some `rm` commands didn't work properly, but the command didn't fail because the `-f` switch ignores errors. The result was that we didn't notice the failures and the docker image became unnecessarily huge. In that Dockerfile we replaced `-f` with `--interactive=never` so that `rm` wouldn't prompt, but still fail on errors.

Reply:
@jheidbrink 

Thx. Nice tip. I will apply it.